From bdf315fc208c8fcf2c1f1cfa5450e51ab9e29f20 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 16 Feb 2008 16:06:55 -0500
Subject: [PATCH] set FAKEROOT to DESTDIR by default

Since the DESTDIR variable is the common standard for installing into a
staging directory, the FAKEROOT variable should default to it.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 Make.Rules |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Make.Rules b/Make.Rules
index db3aab2..9bccbb7 100644
--- a/Make.Rules
+++ b/Make.Rules
@@ -4,7 +4,7 @@
 
 # common 'packaging' directoty
 
-FAKEROOT=
+FAKEROOT=$(DESTDIR)
 
 # Autoconf-style prefixes are activated when $(prefix) is defined.
 # Otherwise binaries and libraraies are installed in /{lib,sbin}/,
-- 
1.5.4

